test/extensions/transport_sockets/tls/context_impl_test.cc

```diff
@@ -42,6 +42,56 @@ namespace Extensions {
 namespace TransportSockets {
 namespace Tls {
 
+namespace {
+const std::vector<std::string> KNOWN_CIPHER_SUITES{"ECDHE-ECDSA-AES128-GCM-SHA256",
```

Comment:
std::vector in global is know to flake tests due to static-initialization fiasco, use CONSTRUCT_ON_FIRST_USE or make it member of the test class (without static)

Reply:
done